iothub/service/src/Messaging/Models/ErrorContext.cs

```diff
@@ -9,52 +10,20 @@ namespace Microsoft.Azure.Devices
     /// The context for a given connection loss event for <see cref="MessageFeedbackProcessorClient"/>,
     /// <see cref="FileUploadNotificationProcessorClient"/>, and <see cref="MessagesClient"/>.
     /// </summary>
-    /// <remarks>
-    /// The context includes the cause of the connection loss and makes a distinction between network
-    /// level issues(no internet) and IoT Hub level issues(resource not found, throttling, internal server error, etc.).
-    /// </remarks>
     public class ErrorContext
```

Comment:
abstract?

I'd prefer we not have an extra class as part of our public API. Maybe we choose not to have a base class?

Reply:
It'll be a small DRY violation, but sure

Reply:
I'll just have 3 concrete classes with no abstract class